apps/src/lib/levelbuilder/shapes.jsx

```diff
@@ -19,9 +19,9 @@ export const levelShape = PropTypes.shape({
   position: PropTypes.number.isRequired,
 
   // level id, or -1 if no level is selected.
-  activeId: PropTypes.number.isRequired,
+  activeId: PropTypes.number,
   // level ids for all variants of this level
-  ids: PropTypes.arrayOf(PropTypes.number).isRequired,
+  ids: PropTypes.arrayOf(PropTypes.number),
```

Comment:
why are these no longer required?

Reply:
I think when you removed the level information from the activities editor this started to give errors. I think we would want this back once level information is being passed in again.

Reply:
Tracking: https://codedotorg.atlassian.net/browse/PLAT-376